library/src/main/java/com/applandeo/materialcalendarview/utils/CalendarProperties.kt

```diff
@@ -151,7 +151,7 @@ class CalendarProperties(private val context: Context) {
     var onPagePrepareListener: OnPagePrepareListener? = null
 
     fun findDayProperties(calendar: Calendar): CalendarDay? =
-            calendarDayProperties.find { it.calendar.isEqual(calendar) }
+            calendarDayProperties.findLast { it.calendar.isEqual(calendar) }
```

Comment:
Why not firstOrNull ?

Reply:
@mateuszSawa Because in this case, we want to display as the first layer the latest configuration of the appearance and not just the first one